src/style.css

```diff
@@ -1,3 +1,111 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+h1 {
+  margin: 20px;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+}
+
+.nav__items {
+  display: block;
+}
+
+.logo {
+  display: block;
+  padding: 10px 0;
+  height: 40px;
+}
+
+.nav__link {
+  display: block;
+  margin: 0 0 0 20px;
+  font-family: Roboto, sans-serif;
```

Comment:
Move to html selector and remove from all other places.

Reply:
+